AegisBuild.mk

```diff
@@ -0,0 +1,61 @@
+#
+# .-'_.---._'-.
+# ||####|(__)||   Protect your secrets, protect your business.
+#   \\()|##//       Secure your sensitive data with Aegis.
+#    \\ |#//                    <aegis.ist>
+#     .\_/.
+#
+
```

Comment:
The tasks build everything and push everything to the registry simultaneously because, more often than not, when you build, you push too. And we can always test things locally before pushing them to the public registry anyway.

If there is a need to build things but not pus, we can add separate targets for those too later.